grakn-migration/json/src/main/java/ai/grakn/migration/json/JsonMigrator.java

```diff
@@ -46,28 +46,30 @@
     private final Set<Reader> readers;
 
     public static void main(String[] args) {
-        MigrationCLI.init(args, JsonMigrationOptions::new).stream()
-                .filter(Optional::isPresent)
-                .map(Optional::get)
-                .forEach(JsonMigrator::runJson);
+        try {
+            MigrationCLI.init(args, JsonMigrationOptions::new).stream()
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .forEach(JsonMigrator::runJson);
+        } catch (Throwable throwable){
+            MigrationCLI.die(throwable.getMessage());
```

Comment:
Hmmm that is a good point. When you migrating data is it all or nothing? I.e. can migration fail midway and only some of the data makes it in? 

If that is the case then we may want to do both, bring the error to the user and log it, so in that case keep the method but rename it because it's certainly not killing anything.